src/pages/Vote/index.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { ComponentMeta } from '@storybook/react';
+import Vote from '.';
+
+export default {
+  title: 'Pages/Vote',
+  component: Vote,
+} as ComponentMeta<typeof Vote>;
+
+export const Default = () => <Vote />;
```

Comment:
We'll need to wrap this story with a `Router` tag:
<img width="463" alt="Screen Shot 2022-05-31 at 9 00 14 AM" src="https://user-images.githubusercontent.com/44675210/171123169-06c6bd32-ad6a-4c4e-a707-59464c9df32c.png">